src/main/kotlin/io/curity/identityserver/plugin/signicat/authentication/SignicatAuthenticatorRequestHandler.kt

```diff
@@ -154,12 +155,14 @@ class SignicatAuthenticatorRequestHandler(config: SignicatAuthenticatorPluginCon
     {
         val taskId = "task_1"
         val request = with(CreateRequestRequest()) {
-            service = serviceName
-            password = useSigning
+            val signingConfiguration = useSigning
                     // For this to throw, the presence of useSigning wasn't checked before. This is a logic error
                     // and should never happen.
                     .orElseThrow { throw exceptionFactory.internalServerException(ErrorCode.PLUGIN_ERROR) }
-                    .secret
+
+            service = serviceName
```

Comment:
maybe it's just the view I'm using here in GitHub to see the changes, but this line and the one below don't seem to have any effect. Am I right?

Reply:
You mean `service` and `password`? They are members of `this` and will be used in the request.

Reply:
Ah, yes. Now I see. Thanks for clarifying.